gn2/wqflask/static/new/javascript/auth/search_mrna.js

```diff
@@ -15,12 +15,17 @@ function search_mrna() {
     selected = JSON.parse(document.getElementById(
 	"tbl-link").getAttribute("data-datasets"));
     species_name = document.getElementById("txt-species-name").value
+    bearer_token = document.getElementById("bearer_token").value
     search_endpoint = "/auth/data/mrna/search"
```

Comment:
@jnduli Rather than expose the bearer token like this, use [this endpoint in GN2](https://github.com/genenetwork/genenetwork2/blob/0998033d0a7ea26ed96b00a360a334bae6de8c55/gn2/wqflask/oauth2/data.py#L107)

Reply:
Done